### PR TITLE
RFXCOM fixes

### PR DESCRIPTION
**Pull request by paulianttila**
- Added missing mandatory parameters to config descriptions.
- Adapted sub type naming to be inline RFXCOM spec.
- Check if bridge is already initialized during sensor initialization. Should prevent sensor to stay stay in offline mode.

Signed-off-by: Pauli Anttila <pauli.anttila@gmail.com>

### CODE REVIEW

**Comment by kaikreuzer** [comment]
Any chance to split "Applied new code formatter for changed files." into a separate commit within this PR? This would make it easier to see the real code changes.

**Comment by paulianttila** [comment]
@kaikreuzer: not sure if you recognised, but PR is without code formatting now.

File: addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/handler/RFXComHandler.java [hunk 1]
```diff
@@ -97,28 +97,38 @@ public void handleCommand(ChannelUID channelUID, Command command) {
...
         logger.debug("Initialized RFXCOM device handler for {}, deviceId={}, subType={}", getThing().getUID(),
                 config.deviceId, config.subType);
+
+        if (config.deviceId == null || config.subType == null) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
+                    "Initialized RFXCOM device missing deviceId or subType");
+        }
+
+        // Check if bridge is already initialized
+        bridgeHandlerInitialized(getBridge().getHandler(), getBridge());
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/handler/RFXComHandler.java]: you must not call this method yourself - it is called by the framework!

File: addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/rain.xml [hunk 2]
```diff
@@ -21,6 +21,26 @@
 			<channel id="batteryLevel" typeId="system.battery-level" />
 			<channel id="lowBattery" typeId="system.low-battery" />
 		</channels>
+		
+		<config-description>
+            <parameter name="deviceId" type="text" required="true">
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/rain.xml]: Can you pls apply the code formatter? You have a mix of spaces and tabs in all XML files now.

**Comment by kaikreuzer** [comment]
> not sure if you recognised, but PR is without code formatting now.

No, I didn't, so thanks for the ping!

**Comment by paulianttila** [comment]
@kaikreuzer Happy?

**Comment by kaikreuzer** [comment]
Yes, thanks, now you only need to squash again!

**Comment by paulianttila** [comment]
I already squashed code changes to one commit, but leaved description file formatting to separated commit. As I reformatted all description files, are you sure, that you want to squash all of them?

**Comment by kaikreuzer** [comment]
ah, ok - that's fine then, thanks a lot!

**Pull request merged**